packages/wallet-sdk/src/provider/CoinbaseWalletProvider.ts

```diff
@@ -56,8 +56,6 @@ export interface CoinbaseWalletProviderOptions {
   relayProvider: () => Promise<WalletSDKRelayAbstract>;
   storage: ScopedLocalStorage;
   diagnosticLogger?: DiagnosticLogger;
-  supportsAddressSwitching?: boolean;
-  isLedger?: boolean;
```

Comment:
isLedger was added from network switching project but actually never used in CBW. removing it now